coops-ui/src/views/AnnualReport.vue

```diff
@@ -286,13 +287,13 @@ export default {
       saveWarnings: [],
 
       // EntityTypes Enum
-      EntityTypes
+      EntityTypes: EntityTypes
```

Comment:
This change is not needed in ES6. See http://es6-features.org/#PropertyShorthand.

Reply:
Good catch, thanks!